<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialWrite</class>
 <widget class="QMainWindow" name="SerialWrite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialWrite</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Input:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="historyLineEdit" name="input" native="true"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_send">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_clear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Output:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="output">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_com">
        <property name="text">
         <string>Com:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="input_com"/>
      </item>
      <item>
       <widget class="QLabel" name="lbl_baud">
        <property name="text">
         <string>Baud:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="input_baud">
        <item>
         <property name="text">
          <string>9600</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>19200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>115200</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_einde">
        <property name="text">
         <string>Einde</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="input_einde">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>\n</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>\r\n</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="input_echo">
        <property name="text">
         <string>Local Echo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btn_Open">
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>output</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>btn_Open</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Configure">
   <property name="text">
    <string>Instellen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>historyLineEdit</class>
   <extends>QWidget</extends>
   <header>historylineedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
